castanet/counter.py

```diff
@@ -16,6 +16,21 @@ def match_imports(cast_dict):
     return imports_dictionary
 
 
+def match_function(cast_dict):
+    """ function to for counting the number of functions in a Python program"""
```

Comment:
Docstring should not have a space between the quotations and the first letter. First letter should be capitalized and the doscstring should end with a period.